components/schema-migrator/migrations/director/202007041321_add_name_labels_to_runtimes.up.sql

```diff
@@ -0,0 +1,10 @@
+
+INSERT INTO labels
+    (id ,tenant_id, runtime_id, "key", "value")
+SELECT uuid_in(md5(random()::text || clock_timestamp()::text)::cstring), unnamed.tenant_id, unnamed.runtime_id, 'name', to_json(unnamed.runtime_name)
```

Comment:
You can use `uuid` extension which should be in PostgreSQL or use pg-crypto
https://www.postgresqltutorial.com/postgresql-uuid/

Reply:
I had to bump psql image as the old one don't have the `uuid-ossp` extension